### ISSUE TITLE
Uncaught RangeError: Can not create an inode from a buffer less than 66 bytes

### ISSUE THREAD

**Issue by rxliuli** [zen-fs/core#135]
Sometimes this issue occurs where certain directories cannot be read, but other directories and files are still normal. I don't know what the specific error is because I can't reproduce it consistently, but it does happen quite easily. After I cleaned the entire idb, it returned to normal.

<img width="857" alt="image" src="https://github.com/user-attachments/assets/57949e1d-d503-4e3a-a1f1-c886752164bf">

```ts
await configure({
  mounts: {
    '/': IndexedDB,
  },
})

Reflect.set(globalThis, '__zenfs__', fs)

console.log('readdir /', await __zenfs__.promises.readdir('/'))
console.log('stat /plugins', await __zenfs__.promises.stat('/plugins'))
console.log('readdir /plugins', await __zenfs__.promises.readdir('/plugins'))
```

**Comment by james-pre** [comment]
@rxliuli, thanks for submitting an issue. If you can create a minimally reproducible example, I can investigate.

One problem may be with 1.3.0's internal change which fixes `Inode` using the `ino` field for a data pointer by renaming the existing field to `data` and adding a new `ino`. Consequently, the size of an `Inode` increased from 58 to 66 bytes, which would cause the error you are having.

**Comment by james-pre** [comment]
Fixed in [1.3.1](https://github.com/zen-fs/core/releases/tag/v1.3.1).
